lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -1449,15 +1449,15 @@ void TypeAnalysis::visitAtenCatOp(AtenCatOp op,
       llvm::map_range(listConstruct.getElements(), [&](Value v) -> ValueKnowledge {
         return getLatticeElement(v)->getValue();
       }));
-  for (auto tensor : tensors) {
-    auto newDtype = meetElementTypes(knowledge.dtype, tensor.dtype);
```

Comment:
The CI says that `meetElementTypes` no longer has users. Can you also remove it?